templates/admin.phtml

```diff
@@ -13,39 +13,49 @@
         <?php require 'partials/layout/nav.phtml'; ?>
```

Comment:
Could you also add a link to the course and add course page to the nav?

SO a new item on the end for Courses, and then the add course page in a dropdown.